mitodl/ocw-hugo-themes
# fix: duplicate IDs

#### Pre-Flight checklist


#### What are the relevant tickets?
- closes https://github.com/mitodl/ocw-hugo-themes/issues/1038
- Just realized, this PR also closes https://github.com/mitodl/ocw-hugo-themes/issues/1070 
    - FYI: @fakhar-ud-din Sorry about this, but if you want me to remove the changes fixing this issue, let me know

#### What's this PR do?
- Fixes all the duplicate IDs

#### How should this be manually tested?
- Checkout this branch
- Build any v2 course
- Verify that there are no duplicate IDs.
    - Duplicate IDs can be found using the JS snippet provided in this [comment](https://github.com/mitodl/ocw-hugo-themes/issues/1038#issuecomment-1420690916) 
- Verify that everything in the course (including CHP, interior pages, etc) works as expected and there are no side effects of these changes.
- Repeat this for course-v3 too
